demo/demo-edge/consumer/src/main/resources/microservice.yaml

```diff
@@ -27,6 +27,9 @@ servicecomb:
     client.http2.useAlpnEnabled: false
   loadbalance:
     filter:
+      service:
+        warmup:
+          enabled: false
```

Comment:
This feature will cause demo-edge fail? Is it an incompatible feature?

Reply:
compatible for edge